ED/protocol.html

```diff
@@ -753,7 +753,8 @@ <h4 property="schema:name">Web Access Control</h4>
                   <section id="auxiliary-resources-description-resource" inlist="" rel="schema:hasPart" resource="#auxiliary-resources-description-resource">
                     <h4 property="schema:name">Description Resource</h4>
                     <div datatype="rdf:HTML" property="schema:description">
-                      <p>An auxiliary resource of type <em>Description Resource</em> provides a description of a subject resource ([<cite><a class="bibref" href="#bib-ldp">LDP</a></cite>]).</p>
+                      <p>An auxiliary resource of type <em>Description Resource</em> provides a description of a subject resource ([<cite><a class="bibref" href="#bib-ldp">LDP</a></cite>]).
+			<span about="" id="server-description-required" rel="spec:requirement" resource="#server-description-required"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> support this auxiliary resource type.</span></span></p>
```

Comment:
And how is it identified? What does supporting it mean?

Reply:
So, this is language pointing at the more generic language in the spec, which also needs work. The `rel` value is defined in a table, the protocol specific things are there too, the lifecycle assumptions also. I recognize it is in many contexts, including this, a bit problematic to just say "support", but I also tend to think that the tests is a venue to provide authoritative interpretations.

This is mostly to fix the immediate omission that `describedby` wasn't required in 0.9.